src/api/constants.ts

```diff
@@ -1,7 +1,26 @@
-export const CLIENT_ID = 'JpwJuEhdlzYrH6FO3kW6QVP8GTr0cVr9';
-export const CLIENT_SECRET =
-  'rctURI8w_fRJe73DPqaI8ASCAbJGCd18p8QsU9nkNpU1_CTadZUzsWzD79YtovLO';
+import { Configuration, ProtocolMode } from '@azure/msal-browser';
+
+export const CLIENT_ID = '7RswiAboRELefuFhVK9RRhaWloTr0VNq';
 export const AUDIENCE = 'https://dev-y4thf51u18ja0ber.eu.auth0.com/api/v2/';
-export const SCOPE = 'openid offline_access';
+export const SCOPE = ['openid', 'offline_access', 'email'];
 export const HEADERS = { 'content-type': 'application/json' };
-export const BASE_URL = 'https://dev-y4thf51u18ja0ber.eu.auth0.com';
\ No newline at end of file
+export const BASE_URL = 'https://dev-y4thf51u18ja0ber.eu.auth0.com';
+
+export const msalConfig: Configuration = {
+  auth: {
+    clientId: CLIENT_ID,
+    authority: BASE_URL,
+    redirectUri: 'http://localhost:3000/home',
```

Comment:
Since all configuration parameters are subjects to change, please move them to .env file.